### PR TITLE
chore(assets): Require scour 0.38.2 (Ubuntu LTS 22.04+) for SVG optimization

### LINKED ISSUE #36014: SVG optimization changes "all" files
Run:
```
bash build/image-optimization.sh
```

Check changed list:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
	modified:   apps/comments/img/comments.svg
	modified:   apps/dashboard/img/dashboard.svg
	modified:   apps/dav/img/schedule.svg
	modified:   apps/encryption/img/app.svg
	modified:   apps/federatedfilesharing/img/social-diaspora.svg
	modified:   apps/federatedfilesharing/img/social-facebook.svg
	modified:   apps/federatedfilesharing/img/social-twitter.svg
	modified:   apps/federation/img/app.svg
	modified:   apps/files/img/add-color.svg
	modified:   apps/files/img/app.svg
	modified:   apps/files/img/change.svg
	modified:   apps/files/img/computer.svg
	modified:   apps/files/img/delete-color.svg
	modified:   apps/files/img/delete.svg
	modified:   apps/files/img/external.svg
	modified:   apps/files/img/folder.svg
	modified:   apps/files/img/public.svg
	modified:   apps/files/img/recent.svg
	modified:   apps/files/img/share.svg
	modified:   apps/files/img/star.svg
	modified:   apps/files_external/img/app-dark.svg
	modified:   apps/files_external/img/app.svg
	modified:   apps/files_sharing/img/app.svg
	modified:   apps/files_trashbin/img/app.svg
	modified:   apps/files_versions/img/app.svg
	modified:   apps/provisioning_api/img/app.svg
	modified:   apps/settings/img/admin.svg
	modified:   apps/settings/img/apps.svg
	modified:   apps/settings/img/change.svg
	modified:   apps/settings/img/help.svg
	modified:   apps/settings/img/password.svg
	modified:   apps/settings/img/personal.svg
	modified:   apps/settings/img/share.svg
	modified:   apps/settings/img/toggle-filelist.svg
	modified:   apps/settings/img/users.svg
	modified:   apps/sharebymail/img/app.svg
	modified:   apps/systemtags/img/app.svg
	modified:   apps/testing/img/app.svg
	modified:   apps/theming/img/accessibility-dark.svg
	modified:   apps/theming/img/app-dark.svg
	modified:   apps/theming/img/app.svg
	modified:   apps/theming/img/default-source.svg
	modified:   apps/updatenotification/img/app.svg
	modified:   apps/updatenotification/img/notification.svg
	modified:   apps/user_ldap/img/app-dark.svg
	modified:   apps/user_ldap/img/app.svg
	modified:   apps/user_ldap/img/copy.svg
	modified:   apps/user_status/img/app-dark.svg
	modified:   apps/user_status/img/app.svg
	modified:   apps/weather_status/img/app-dark.svg
	modified:   apps/weather_status/img/app.svg
	modified:   apps/weather_status/img/cloud-cloud.svg
	modified:   apps/weather_status/img/cloud-dots.svg
	modified:   apps/weather_status/img/cross.svg
	modified:   apps/weather_status/img/drop.svg
	modified:   apps/weather_status/img/fog.svg
	modified:   apps/weather_status/img/half-sun.svg
	modified:   apps/weather_status/img/heavy-rain.svg
	modified:   apps/weather_status/img/light-rain.svg
	modified:   apps/weather_status/img/moon-cloud-heavy-rain.svg
	modified:   apps/weather_status/img/moon-cloud-light-rain.svg
	modified:   apps/weather_status/img/moon-cloud-rain.svg
	modified:   apps/weather_status/img/moon-cloud.svg
	modified:   apps/weather_status/img/moon-small-cloud.svg
	modified:   apps/weather_status/img/moon.svg
	modified:   apps/weather_status/img/rain.svg
	modified:   apps/weather_status/img/snow.svg
	modified:   apps/weather_status/img/sun-cloud-heavy-rain.svg
	modified:   apps/weather_status/img/sun-cloud-light-rain.svg
	modified:   apps/weather_status/img/sun-cloud-rain.svg
	modified:   apps/weather_status/img/sun-cloud.svg
	modified:   apps/weather_status/img/sun-small-cloud.svg
	modified:   apps/weather_status/img/sun.svg
	modified:   apps/weather_status/img/thunder.svg
	modified:   apps/weather_status/img/umbrella.svg
	modified:   core/img/actions/add.svg
	modified:   core/img/actions/address.svg
	modified:   core/img/actions/alert-outline.svg
	modified:   core/img/actions/arrow-left.svg
	modified:   core/img/actions/arrow-right.svg
	modified:   core/img/actions/audio-off.svg
	modified:   core/img/actions/audio.svg
	modified:   core/img/actions/caret-white.svg
	modified:   core/img/actions/caret.svg
	modified:   core/img/actions/change.svg
	modified:   core/img/actions/checkbox-mark-dark.svg
	modified:   core/img/actions/checkbox-mark-white.svg
	modified:   core/img/actions/checkbox-mark.svg
	modified:   core/img/actions/checkbox-mixed-dark.svg
	modified:   core/img/actions/checkbox-mixed-white.svg
	modified:   core/img/actions/checkbox-mixed.svg
	modified:   core/img/actions/checkmark-white.svg
	modified:   core/img/actions/checkmark.svg
	modified:   core/img/actions/clippy.svg
	modified:   core/img/actions/close.svg
	modified:   core/img/actions/comment.svg
	modified:   core/img/actions/confirm-fade.svg
	modified:   core/img/actions/confirm-white.svg
	modified:   core/img/actions/confirm.svg
	modified:   core/img/actions/delete.svg
	modified:   core/img/actions/download.svg
	modified:   core/img/actions/edit.svg
	modified:   core/img/actions/error-white.svg
	modified:   core/img/actions/error.svg
	modified:   core/img/actions/external.svg
	modified:   core/img/actions/filter.svg
	modified:   core/img/actions/fullscreen.svg
	modified:   core/img/actions/group.svg
	modified:   core/img/actions/history.svg
	modified:   core/img/actions/logout.svg
	modified:   core/img/actions/mail.svg
	modified:   core/img/actions/mastodon.svg
	modified:   core/img/actions/menu-sidebar.svg
	modified:   core/img/actions/menu.svg
	modified:   core/img/actions/more-white.svg
	modified:   core/img/actions/more.svg
	modified:   core/img/actions/password-white.svg
	modified:   core/img/actions/password.svg
	modified:   core/img/actions/pause.svg
	modified:   core/img/actions/phone.svg
	modified:   core/img/actions/play-add.svg
	modified:   core/img/actions/play-next.svg
	modified:   core/img/actions/play-previous.svg
	modified:   core/img/actions/play.svg
	modified:   core/img/actions/profile.svg
	modified:   core/img/actions/projects.svg
	modified:   core/img/actions/public-white.svg
	modified:   core/img/actions/public.svg
	modified:   core/img/actions/quota.svg
	modified:   core/img/actions/recent.svg
	modified:   core/img/actions/rename.svg
	modified:   core/img/actions/screen-off.svg
	modified:   core/img/actions/screen.svg
	modified:   core/img/actions/search.svg
	modified:   core/img/actions/settings-dark.svg
	modified:   core/img/actions/settings.svg
	modified:   core/img/actions/share.svg
	modified:   core/img/actions/shared.svg
	modified:   core/img/actions/sound-off.svg
	modified:   core/img/actions/sound.svg
	modified:   core/img/actions/star-dark.svg
	modified:   core/img/actions/star.svg
	modified:   core/img/actions/starred.svg
	modified:   core/img/actions/tag.svg
	modified:   core/img/actions/template-add.svg
	modified:   core/img/actions/timezone.svg
	modified:   core/img/actions/toggle-background.svg
	modified:   core/img/actions/toggle-filelist.svg
	modified:   core/img/actions/toggle-pictures.svg
	modified:   core/img/actions/toggle.svg
	modified:   core/img/actions/triangle-e.svg
	modified:   core/img/actions/triangle-n.svg
	modified:   core/img/actions/triangle-s.svg
	modified:   core/img/actions/twitter.svg
	modified:   core/img/actions/upload.svg
	modified:   core/img/actions/user.svg
	modified:   core/img/actions/verified.svg
	modified:   core/img/actions/verify.svg
	modified:   core/img/actions/verifying.svg
	modified:   core/img/actions/video-off.svg
	modified:   core/img/actions/video-switch.svg
	modified:   core/img/actions/video.svg
	modified:   core/img/actions/view-close.svg
	modified:   core/img/actions/view-download.svg
	modified:   core/img/actions/view-next.svg
	modified:   core/img/actions/view-pause.svg
	modified:   core/img/actions/view-play.svg
	modified:   core/img/actions/view-previous.svg
	modified:   core/img/apps/circles.svg
	modified:   core/img/apps/richdocuments.svg
	modified:   core/img/apps/richdocumentscode.svg
	modified:   core/img/appstore.svg
	modified:   core/img/breadcrumb.svg
	modified:   core/img/caldav/attendees.svg
	modified:   core/img/caldav/description.svg
	modified:   core/img/caldav/link.svg
	modified:   core/img/caldav/location.svg
	modified:   core/img/caldav/organizer.svg
	modified:   core/img/caldav/time.svg
	modified:   core/img/caldav/title.svg
	modified:   core/img/categories/auth.svg
	modified:   core/img/categories/bundles.svg
	modified:   core/img/categories/customization.svg
	modified:   core/img/categories/dashboard.svg
	modified:   core/img/categories/files.svg
	modified:   core/img/categories/games.svg
	modified:   core/img/categories/integration.svg
	modified:   core/img/categories/monitoring.svg
	modified:   core/img/categories/multimedia.svg
	modified:   core/img/categories/office.svg
	modified:   core/img/categories/organization.svg
	modified:   core/img/categories/social.svg
	modified:   core/img/clients/desktop.svg
	modified:   core/img/clients/phone.svg
	modified:   core/img/clients/tablet.svg
	modified:   core/img/desktopapp.svg
	modified:   core/img/f-droid.svg
	modified:   core/img/facebook.svg
	modified:   core/img/favicon-mask.svg
	modified:   core/img/favicon-touch.svg
	modified:   core/img/favicon.svg
	modified:   core/img/filetypes/application-pdf.svg
	modified:   core/img/filetypes/application.svg
	modified:   core/img/filetypes/audio.svg
	modified:   core/img/filetypes/file.svg
	modified:   core/img/filetypes/folder-drag-accept.svg
	modified:   core/img/filetypes/folder-encrypted.svg
	modified:   core/img/filetypes/folder-external.svg
	modified:   core/img/filetypes/folder-public.svg
	modified:   core/img/filetypes/folder-shared.svg
	modified:   core/img/filetypes/folder-starred.svg
	modified:   core/img/filetypes/folder.svg
	modified:   core/img/filetypes/image.svg
	modified:   core/img/filetypes/link.svg
	modified:   core/img/filetypes/location.svg
	modified:   core/img/filetypes/mindmap.svg
	modified:   core/img/filetypes/package-x-generic.svg
	modified:   core/img/filetypes/text-calendar.svg
	modified:   core/img/filetypes/text-code.svg
	modified:   core/img/filetypes/text-vcard.svg
	modified:   core/img/filetypes/text.svg
	modified:   core/img/filetypes/video.svg
	modified:   core/img/filetypes/x-office-document.svg
	modified:   core/img/filetypes/x-office-drawing.svg
	modified:   core/img/filetypes/x-office-form-template.svg
	modified:   core/img/filetypes/x-office-form.svg
	modified:   core/img/filetypes/x-office-presentation.svg
	modified:   core/img/filetypes/x-office-spreadsheet.svg
	modified:   core/img/logo/logo-enterprise.svg
	modified:   core/img/mail.svg
	modified:   core/img/mastodon.svg
	modified:   core/img/places/calendar.svg
	modified:   core/img/places/contacts.svg
	modified:   core/img/places/default-app-icon.svg
	modified:   core/img/places/files.svg
	modified:   core/img/places/home.svg
	modified:   core/img/places/link.svg
	modified:   core/img/places/music.svg
	modified:   core/img/places/picture.svg
	modified:   core/img/rating/s0.svg
	modified:   core/img/rating/s1.svg
	modified:   core/img/rating/s10.svg
	modified:   core/img/rating/s2.svg
	modified:   core/img/rating/s3.svg
	modified:   core/img/rating/s4.svg
	modified:   core/img/rating/s5.svg
	modified:   core/img/rating/s6.svg
	modified:   core/img/rating/s7.svg
	modified:   core/img/rating/s8.svg
	modified:   core/img/rating/s9.svg
	modified:   core/img/rss.svg
	modified:   core/img/twitter.svg
	modified:   themes/example/core/img/favicon-touch.svg
	modified:   themes/example/core/img/favicon.svg
	modified:   themes/example/core/img/logo-icon.svg
	modified:   themes/example/core/img/logo.svg
```

If a specific version is required we should check for it.

### PR DESCRIPTION
**Pull request by nickvergessen**
* Resolves: #36014

## Summary

* Pin scour version minimum
* Optimize all SVGs
* Optimize new PNGs and JPGs

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: cypress/fixtures/image.jpg
Comment (jancborchardt) [inline cypress/fixtures/image.jpg:1]: Is this something we should touch? Or should this also be excluded/ignored?

File: build/integration/data/coloured-pattern-non-square.png
Comment (jancborchardt) [inline build/integration/data/coloured-pattern-non-square.png:1]: This we should probably exclude as well?

File: build/integration/data/green-square-256.png
Comment (jancborchardt) [inline build/integration/data/green-square-256.png:1]: This we should probably exclude as well?

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Added comments on 3 images I’m not sure we should touch – otherwise it looks good! :)

**Review by come-nc — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
Nit: squash the commits into one to keep our git history light

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**